services/brig/test/integration/Federation/End2end.hs

```diff
@@ -183,14 +183,14 @@ testGetUsersById brig1 brig2 = do
 
 testClaimPrekeySuccess :: Brig -> Brig -> Http ()
 testClaimPrekeySuccess brig1 brig2 = do
-  self <- randomUser brig1
-  user <- randomUser brig2
+  self <- randomUser brig2
+  user <- randomUser brig1
```

Comment:
Out of curiosity, would it still work if the two brigs are switched back?

Reply:
It should, this is just a leftover. Thanks for catching.